### ISSUE TITLE
Bug: RPP Edit Payment Info

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#5959]
when we are in an application and trying to put in payment information the ‘edit payment info’ does not work. 
When clicked on it backs you out of the whole application

I was able to duplicate this issue.

**Comment by ChrispinP** [comment]
I have identified this as a bug, the entire Item Properties for the action link has been wiped. I have reported to Knack. I do not want to remove the link just in case it might also remove the edit payment info child page attached to it.
![image](https://user-images.githubusercontent.com/67915167/116727299-fb8fbb00-a9a9-11eb-8db2-c771e5cfc455.png)


Current workaround is to only use the Edit Payment menu button at the top of the page for now
![image](https://user-images.githubusercontent.com/67915167/116727281-f599da00-a9a9-11eb-8f42-3f29e9b3a6fa.png)
